torch/_inductor/kernel/flex_attention.py

```diff
@@ -138,11 +138,7 @@ def maybe_realize(args: list[Optional[IRNode]]):
 
 
 def get_float32_precision():
-    if (
-        torch.get_float32_matmul_precision() == "highest"
-        or torch.version.hip
-        or torch.mtia.is_available()
-    ):
+    if torch.get_float32_matmul_precision() == "highest" or torch.version.hip:
```

Comment:
rebase bug?

Reply:
yeah looks like it; updated

Reply:
can you just revert this change entirely, still seems like the MTIA change is gone